Added code to resize columns to fit data.

cienvaras opened the pull request:
Fix for this: https://github.com/NuCivic/ok_data/issues/134

topicus commented on js/visualization_entity_tables_edit.js:
```diff
@@ -1,13 +1,25 @@
       var re = /\[(.*?)\]/;
       var uuid = re.exec($resourceField.val())[1];
-      $('#edit-field-ve-settings-und-0-value').val(JSON.stringify({source:'/node/' + uuid + '/download'}));
+      settings.source = '/oklahoma/node/' + uuid + '/download';
```
We need to remove this before merge to prevent this library remain tied to the OK project.

cienvaras replied:
Ok, fixed.

topicus commented:
Could you provide a briefly explanation with the steps I need to follow in order to make this work? I've cloned and switched to this branch but after that I couldn't see any option in the table's creation page.

However I've got an error in the chrome console: 
add:1 Uncaught SyntaxError: Unexpected end of input

I did a drush fr visualization_entity_tables just in case would be an eck update but without succes.

cienvaras commented:
@topicus I just added a new field to the bundle and removed some unused elements, it should work with a simple feature revert.

I can't reproduce the error you're getting. It might be related with the checkbox not being added to the bundle.

topicus commented on js/visualization_entity_tables_edit.js:
```diff
@@ -1,13 +1,25 @@
+
   $(document).on('ready', function(){
+
+    var settings = JSON.parse($('#edit-field-ve-settings-und-0-value').val());
```
Here you are assuming that $('#edit-field-ve-settings-und-0-value').val() is an object, that was causing the end of input error. You need to check that before the parse.

cienvaras replied:
Added exception handling to prevent this error.
